src/model/orders/Order.java

```diff
@@ -0,0 +1,72 @@
+package model.orders;
+
+import model.Product;
+import model.persons.Client;
+import java.util.HashMap;
+import java.util.Map;
+
+import java.util.Collection;
+
+/**
+ * The Order Class has this Client and the list of OrderDetail (list of product and quantity)
+ */
+public class Order {
+    Integer id;
+    Client client;
+    Collection<OrderDetail> orderDetailList;
+    String note;
+
+    /**
+     * @return The price of the list of products
+     */
+    protected Double getProductListPrice(){
+        Double totalPrice = 0.0;
+        for (OrderDetail orderDetail: orderDetailList) {
+            totalPrice+=orderDetail.getProduct().getPrice();
+        }
+        return totalPrice;
+    }
+
+    /**
+     * @return this method calculates the estimated preparation time of the whole order
+     */
+    public Double getEstimatedPreparationTime(){
+        return null;
+    }
+
+    public Integer getId() {
```

Comment:
Id should be long

Reply:
Woulg you tell me why it shoul be **Long** so I know if a have to change the **id** field in all the other model classes?

Because I am using **Integer** in all aproved model classes, I am even using int in some classes.